kubernetes-client-api/src/main/java/io/fabric8/kubernetes/client/http/HttpRequest.java

```diff
@@ -55,13 +55,19 @@ default Builder put(String contentType, String writeValueAsString) {
       return method("PUT", contentType, writeValueAsString);
     }
 
+    default Builder put(String contentType, InputStream stream, long length) {
+      return method("PUT", contentType, stream, length);
+    }
+
     default Builder post(String contentType, String writeValueAsString) {
       return method("POST", contentType, writeValueAsString);
     }
 
     Builder post(String contentType, byte[] writeValueAsBytes);
 
-    Builder post(String contentType, InputStream stream, long length);
+    default Builder post(String contentType, InputStream stream, long length) {
```

Comment:
Since we are providing a default implementation, we want to remove any of the other implementations (See StandardHttpClient) which should probably be covered by this one.

Reply:
yes

Reply:
I've removed post implementation from StandardHttpRequest.